http/handlers.go

```diff
@@ -1,32 +1,35 @@
 package http
 
 import (
-	"encoding/json"
+	"encoding/gob"
+	"fmt"
+	"io"
 	"net/http"
 
-	"github.com/benchttp/server/internal"
+	"github.com/benchttp/server"
 )
 
 func (s *Server) handlePostReport(rw http.ResponseWriter, r *http.Request) {
-	report := internal.Report{}
-	if err := json.NewDecoder(r.Body).Decode(&report); err != nil {
+	o := server.Output{}
+
+	err := gob.NewDecoder(r.Body).Decode(&o)
+	if err != nil && err != io.EOF {
```

Comment:
```suggestion
	if err != nil {
```
(`Decode`'s `io.EOF` is not expected)